lib/galaxy/webapps/galaxy/api/job_lock.py

```diff
@@ -11,12 +11,12 @@
 
 
 @router.get('/api/job_lock')
-def job_lock_status(job_manager=Depends(get_job_manager), admin_user=Depends(get_admin_user)) -> JobLock:
+async def job_lock_status(job_manager=Depends(get_job_manager), admin_user=Depends(get_admin_user)) -> JobLock:
     """Get job lock status."""
     return job_manager.job_lock()
 
 
 @router.put('/api/job_lock')
-def update_job_lock(job_manager=Depends(get_job_manager), admin_user=Depends(get_admin_user), job_lock: JobLock = Body(...)) -> JobLock:
+async def update_job_lock(job_manager=Depends(get_job_manager), admin_user=Depends(get_admin_user), job_lock: JobLock = Body(...)) -> JobLock:
```

Comment:
These use the ORM, they should not be async routes.

Reply:
Apologies - https://github.com/galaxyproject/galaxy/pull/11128